packages/fiori/src/ShellBar.ts

```diff
@@ -459,10 +170,363 @@ const metadata = {
+	 *
+	 * <ul>
+	 * 		<li><code>expanded</code>: Indicates whether the anchor element, or another grouping element it controls, is currently expanded or collapsed. Accepts the following string values:
+	 *			<ul>
+	 *				<li><code>true</code></li>
+	 *				<li><code>false</code></li>
+	 *			</ul>
+	 * 		</li>
+	 * 		<li><code>hasPopup</code>: Indicates the availability and type of interactive popup element, such as menu or dialog, that can be triggered by the anchor element. Accepts the following string values:
+	 * 			<ul>
+	 *				<li><code>Dialog</code></li>
+	 *				<li><code>Grid</code></li>
+	 *				<li><code>ListBox</code></li>
+	 *				<li><code>Menu</code></li>
+	 *				<li><code>Tree</code></li>
+	 * 			</ul>
+	 * 		</li>
+	 * </ul>
+	 * @type {object}
+	 * @name sap.ui.webc.main.ShellBar.prototype.accessibilityAttributes
+	 * @public
+	 * @since 1.10.0
+	 */
+	 @property({ type: Object })
+	 accessibilityAttributes!: AccessibilityAttributes;
+
+	/**
+	 * @private
+	 */
+	@property()
+	breakpointSize!: string;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	coPilotActive!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	withLogo!: boolean;
+
+	@property({ type: Object })
+	_itemsInfo!: Array<IShelBarItemInfo>;
+
+	@property({ type: Object })
+	_header!: object;
+
+	@property({ type: Object, multiple: true })
+	_menuPopoverItems!: Array<HTMLElement>;
+
+	@property({ type: Boolean, noAttribute: true })
+	_menuPopoverExpanded!: boolean;
+
+	@property({ type: Boolean, noAttribute: true })
+	_overflowPopoverExpanded!: boolean;
+
+	@property({ type: Boolean, noAttribute: true })
+	_fullWidthSearch!: boolean;
+
+	/**
+	 * Defines the <code>ui5-shellbar</code> aditional items.
+	 * <br><br>
+	 * <b>Note:</b>
+	 * You can use the &nbsp;&lt;ui5-shellbar-item>&lt;/ui5-shellbar-item>.
+	 *
+	 * @type {sap.ui.webc.fiori.IShellBarItem[]}
+	 * @name sap.ui.webc.main.ShellBar.prototype.default
+	 * @slot items
+	 * @public
+	 */
+	@slot({ type: HTMLElement, "default": true, invalidateOnChildChange: true })
+	items!: Array<ShellBarItem>;
+
+	/**
+	 * You can pass <code>ui5-avatar</code> to set the profile image/icon.
+	 * If no profile slot is set - profile will be excluded from actions.
+	 *
+	 * Note: We recommend not using the <code>size</code> attribute of <code>ui5-avatar</code> because
+	 * it should have specific size by design in the context of <code>ui5-shellbar</code> profile.
+	 * @type {sap.ui.webc.main.IAvatar}
+	 * @name sap.ui.webc.main.ShellBar.prototype.profile
+	 * @slot profile
+	 * @since 1.0.0-rc.6
+	 * @public
+	 */
+	@slot({ type: HTMLElement })
+	profile!: Array<HTMLElement>;
+
+	/**
+	 * Defines the logo of the <code>ui5-shellbar</code>.
+	 * For example, you can use <code>ui5-avatar</code> or <code>img</code> elements as logo.
+	 * @type {sap.ui.webc.main.IAvatar}
+	 * @name sap.ui.webc.main.ShellBar.prototype.logo
+	 * @slot
+	 * @since 1.0.0-rc.8
+	 * @public
+	 */
+	@slot({ type: HTMLElement })
```

Comment:
`@slot({ type: HTMLElement })` is the default value of the slot decorator, so you can just write `@slot()`
here and for the next 3 slots as well